addon/components/virtual-each.js

```diff
@@ -178,7 +191,7 @@ const VirtualEachComponent = Component.extend(EventListenerMixin, DefaultAttrsMi
         _totalHeight: Math.max(get(items, 'length') * this.getAttr('itemHeight'), 0)
       });
 
-      if (attrs.newAttrs.hasOwnProperty('items') || attrs.newAttrs.hasOwnProperty('positionIndex')) {
```

Comment:
We do not ever scrollTo when items change, because this is handled by observers on `_items`